Habinezajanvier/Anes_Pro
# Setting environment variables

#### What does this PR do?
Setting up development environment
#### Description of Task to be completed?
* setting up express
*  mongodb entegration
* setting mongoose
#### How should this be manually tested?
* clone the repo
* check out the branch
* create a user in mongo
```javascript
db.createUser(
  {
    user: "sammy",
    pwd: "your_password",
    roles: [ { role: "userAdminAnyDatabase", db: "admin" } ]
  }
)
````
* Create `.env` files with the credentials
```javascript
DB_USER_DEV=username
DB_PASSWORD_DEV=password
DB_HOST_DEV=hostName
DB_PORT_DEV=portName
DB_NAME_DEV=databaseName
```
#### Any background context you want to provide?
You have to install mongodb first, in order to run all these above
#### What are the relevant pivotal tracker stories?
#172283759
